File: Lista 1/five/5.py
fuel_efficiency = 12  # km per liter
average_speed = float(input("Enter the average speed of the trip (km/h): "))
time_spent = float(input("Enter the time spent on the trip (hours): "))
distance_traveled = average_speed * time_spent
liters_used = distance_traveled / fuel_efficiency

print(f"\nTrip Results:")
print(f"Average speed: {average_speed} km/h")
print(f"Time spent: {time_spent} hours")
print(f"Distance traveled: {distance_traveled} km")
print(f"Fuel used: {liters_used:.2f} liters")

File: Lista 1/5/5.c
#include <stdio.h>

int main(){

    int fuelEfficiency = 12;
    double avaregeSpeed, distanceTraveled, litersUsed;
    int timeSpent;


    printf("Enter the average speed of the trip (km/h):\n");
    scanf("%lf", &avaregeSpeed);
    getchar();
    printf("Enter the time spent on the trip (hours):\n");
    scanf("%d", &timeSpent);

    distanceTraveled = avaregeSpeed * (double)timeSpent;
    litersUsed = distanceTraveled / (double)fuelEfficiency;

    printf("\nTrip Results:");
    printf("Average speed: %.2lf km/h\n", avaregeSpeed);
    printf("Time spent: %d hours\n", timeSpent);
    printf("Distance traveled: %.2lf km\n", distanceTraveled );
    printf("Fuel used: %.2lf liters\n", litersUsed);

    return 0;
}